README.md

```diff
@@ -4,6 +4,10 @@
 
 _[The code for these tutorials (and this site) can be found here on Github](https://www.github.com/cosmos/sdk-tutorials)_
 
+## Prerequisites
+
+After a quick [Development Environment Setup](SETUP.md), you're ready to select a tutorial from the list below.
```

Comment:
I think it may be better to link to the official go download page instead of providing a doc that says how to install it.

Reply:
No way. Setup.md contains extracted explanations (the overall task took me hours), some specific to the tutorials (e.g. windows status).

Linking simply to a download page is sloppy.

Reply:
we have had this discussion multiple times. The decision was to link to the go install page. this is how it is done in Tendermint, Gaia and the Cosmos-sdk.

Reply:
Well, if you're all proud of your result, then simply shred this PR.

Reply:
I definitely there is a way we can work together to get this pull request merged

Reply:
see https://github.com/cosmos/sdk-tutorials/pull/238#discussion_r370647498